app/contact.html

```diff
@@ -0,0 +1,80 @@
+{% extends 'base.njk' %}
+
+{% block title %}$( contact.title )$ | Bitcoin Cash Node{% endblock %}
+{% block description %}{% endblock %}
+
+{% block body %}
+<section class="bg-dark">
+  <div class="container">
+    <div class="row pb-3">
+      <div class="col py-5 text-center text-md-left">
+        <h1 class="hero-title">$( contact.title )$</h1>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="bg-light">
+  <div class="container py-5">
+
+    <div class="row">
+      <!-- Support -->
+      <div class="col-6 col-md-3 mb-4 mb-md-0">
+        <div class="h3">{% include "icons/chat.svg" %}</div>
+        <div class="border-left-lg pl-3 mt-4">
+          <div class="h3">$( contact.support )$</div>
+          <ul class="list-inline social">
+            <li>
+              <a class="text-muted" target="_blank" href="https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/">GitLab</a>
```

Comment:
I would suggest making this link directly to the Issues list at

https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/-/issues/

Users looking for support are going to be confused if they get presented with the general repository overview/activities/MRs etc. which aren't relevant to them.